src/basic/missing.h

```diff
@@ -68,14 +67,6 @@
 #define NS_GET_NSTYPE _IO(0xb7, 0x3)
 #endif
 
-#ifndef FALLOC_FL_KEEP_SIZE
-#define FALLOC_FL_KEEP_SIZE 0x01
-#endif
-
-#ifndef FALLOC_FL_PUNCH_HOLE
-#define FALLOC_FL_PUNCH_HOLE 0x02
```

Comment:
Isn't this still used in fs-util.c?

Reply:
Similar to the others, which has been provided earlier than kernel-3.11.